src/components/CalendarMonth.jsx

```diff
@@ -197,7 +197,9 @@ class CalendarMonth extends React.PureComponent {
           )}
         >
           {renderMonthElement ? (
-            renderMonthElement({ month, onMonthSelect, onYearSelect })
+            renderMonthElement({
+              month, onMonthSelect, onYearSelect, isVisible,
```

Comment:
If the entire object literal cannot fit on one line, each object property here must be on its own line 
```suggestion
              month,
              onMonthSelect,
              onYearSelect,
              isVisible,
```